powersimdata/utility/templates/ScenarioList.csv

```diff
@@ -1 +1 @@
-id,plan,name,state,interconnect,base_demand,base_hydro,base_solar,base_wind,change_table,start_date,end_date,interval,engine,runtime,infeasibilities
+id,plan,name,state,interconnect,base_demand,base_hydro,base_solar,base_wind,change_table,start_date,end_date,interval,engine,runtime,infeasibilities,grid_model
```

Comment:
Should the new `grid_model` column live next to the `interconnect` column? I think we factored out all the hard-coded column numbers, so changing the position of the other columns shouldn't matter.

Reply:
As you wish. I will have to switch the order in the Create class.

Reply:
Done